--- FindMyApp service.wadl.orig	2011-07-11 00:14:59.000000000 +0200
+++ FindMyApp service.wadl	2011-07-11 00:17:13.585090978 +0200
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="wadl.xsl"?>
 <application xmlns="http://wadl.dev.java.net/2009/02">
   <doc xml:lang="en" title="FindMyApp service">UKA-11 student festival service</doc>
   <resources base="http://findmyapp.net/findmyapp">
