--- FindMyApp service.wadl.orig	Thu Jul  7 15:03:21 2011
+++ FindMyApp service.wadl	Thu Jul  7 15:04:39 2011
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="wadl.xsl"?>
 <application xmlns="http://wadl.dev.java.net/2009/02">
   <doc xml:lang="en" title="FindMyApp service">UKA-11 student festival service</doc>
   <resources base="http://findmyapp.net/findmyapp">
